src/Integration/WPF/ContextualCommandViewModel.cs

```diff
@@ -43,18 +44,29 @@ public class ContextualCommandViewModel : ViewModelBase
         /// <param name="fixedContext">Required context</param>
         /// <param name="command">Optional real command to trigger and pass the fixed context to</param>
         public ContextualCommandViewModel(object fixedContext, ICommand command)
+            : this(fixedContext, command, commandArgs: null)
+        {
+        }
+
+        /// <summary>
+        /// Creates an instance of contextual command view model
+        /// </summary>
+        /// <param name="fixedContext">Required context</param>
+        /// <param name="command">Optional real command to trigger and pass the fixed context to</param>
+        /// <param name="commandArgs">Optional arguments to pass to the command. If null then <paramref name="fixedContent"/> will be passed.</param>
+        public ContextualCommandViewModel(object fixedContext, ICommand command, object commandArgs)
```

Comment:
This class is used for the context menus and clickable notifications in the Team Explorer. The command that is executed is fixed, but it's possible for the text and icon to be set dynamically.

Previously, the class assumed that the  _fixedContext_ would be used to both dynamically update the UI, and to be the argument to pass to the command. The changes to the binding configuration mean this is no longer true.